owncloudApp/src/main/java/com/owncloud/android/MainApp.kt

```diff
@@ -133,24 +135,30 @@ class MainApp : Application() {
                         ReleaseNotesActivity.runIfNeeded(activity)
 
                         val pref = PreferenceManager.getDefaultSharedPreferences(appContext)
-                        val dontShowAgainDialogPref = pref.getBoolean(PREFERENCE_KEY_DONT_SHOW_OCIS_ACCOUNT_WARNING_DIALOG, false)
-                        if (!dontShowAgainDialogPref && shouldShowDialog(activity)) {
-                            val checkboxDialog = activity.layoutInflater.inflate(R.layout.checkbox_dialog, null)
-                            val checkbox = checkboxDialog.findViewById<CheckBox>(R.id.checkbox_dialog)
-                            checkbox.setText(R.string.ocis_accounts_warning_checkbox_message)
-                            val builder = AlertDialog.Builder(activity).apply {
-                                setView(checkboxDialog)
-                                setTitle(R.string.ocis_accounts_warning_title)
-                                setMessage(R.string.ocis_accounts_warning_message)
-                                setCancelable(false)
-                                setPositiveButton(R.string.ocis_accounts_warning_button) { _, _ ->
-                                    if (checkbox.isChecked) {
-                                        pref.edit().putBoolean(PREFERENCE_KEY_DONT_SHOW_OCIS_ACCOUNT_WARNING_DIALOG, true).apply()
+                        val isClearDataPrefAvailable = pref.contains(PREFERENCE_CLEAR_DATA)
```

Comment:
This name doesn't indicate what does the fact that the `PREFERENCE_CLEAR_DATA` exists mean, will be harder for the future to understand it

Reply:
`PREFERENCE_CLEAR_DATA` is not suitable at all either, what would you think that a preference called `PREFERENCE_CLEAR_DATA` set to `true` means?

Reply:
Changed